### ISSUE TITLE
Bug with images using dev server

### ISSUE THREAD

**Issue by dx1ded** [jantimon/html-webpack-plugin#1714]
## Current behaviour 💣

Hello! I tried to use [image-minimizer-webpack-plugin](https://webpack.js.org/plugins/image-minimizer-webpack-plugin/) in dev server but there's something strange. I have generators (for `.webp` and `.avif`) and i try to require it in HTML (in my case it is `.pug` but never mind). When i have one require it is ok, but when i have two requires or more - the last one generates image, each previous returns default value (in my case it is `.jpg`).

## Expected behaviour ☀️

All requires will generate expected file independently of count

### Reproduction Example 👾

- [Video demonstration](https://www.youtube.com/watch?v=l1Q6V9ELIdk&ab_channel=VolodymyrDoskochinskiy)
- [Webpack configuration](https://github.com/dx1ded/webpack-mpa/blob/main/webpack.config.js)

## Environment 🖥

- **nodejs** - `v15.9.0`
- **npm** - `v7.6.3`
- **webpack** - `v5.65.0`
- **webpack-cli** - `v4.9.1`
- **webpack-dev-server** - `v4.7.0`

**Comment by jantimon** [comment]
Unfortunately I don't know that image minification plugin but if it doesn't work well you might open an issue for it.

Did you try to use an image minification loader instead?

**Comment by dx1ded** [comment]
@jantimon, no, it works greatly but when i open dev server there's some problems which i explained upper. I have already opened an issue in `image-minimizer-webpack-plugin` repository but they asked me that the problem lies in `html-webpack-plugin`. Speaking about another image minification loader - no, i haven't tried to use another one.

**Comment by alexander-akait** [comment]
@dx1ded big sorry for delay, do you still a such problem? can you provide reproducible example?
